staker-rewards/lib/rewards-calculator.js

```diff
@@ -108,6 +121,21 @@ export default class RewardsCalculator {
     return new BigNumber(minimumStake)
   }
 
+  async isUndelegatingOperator(operatorAddress) {
+    const block = this.interval.startBlock
+
+    const tokenStaking = await this.context.contracts.TokenStaking.deployed()
+
+    const delegationInfo = await callWithRetry(
+      tokenStaking.methods.getDelegationInfo(operatorAddress),
+      block
+    )
+
+    const undelegatedAt = new BigNumber(delegationInfo.undelegatedAt).toNumber()
```

Comment:
Why do we need to do this transformation? I am not fully following it.

Reply:
Yeah, we can directly compare to `"0"` because we are no longer checking the `undelegatedAt` against interval start timestamp.